python/cudf/dataframe/dataframe.py

```diff
@@ -294,9 +294,13 @@ def __len__(self):
         return self._size
 
     def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
-        if method == '__call__' and 'sqrt' == ufunc.__name__:
-            from cudf import sqrt
-            return sqrt(self)
+        if (method == '__call__' and
+                ufunc.__name__ in ['sqrt', 'sin', 'cos', 'tan',
+                                   'arcsin', 'arccos', 'arctan',
+                                   'exp', 'log']):
```

Comment:
We should look to generalize this in the future with something like `hasattr(cudf, ufunc.__name__)` but this is fine for now.

Reply:
@devavret why not go ahead and do this in this PR? Then we can merge.

Reply:
Gladly, @kkraus14 , can you explain a but more what you’d like?